multimodal/setup.py

```diff
@@ -28,6 +28,8 @@
     "tqdm",
     "boto3",
     "requests",
+    "evaluate",
```

Comment:
We may use "evaluate<=0.2.2"

Reply:
Done in this [commit](https://github.com/awslabs/autogluon/pull/2183/commits/ec2fc468bf390caee74616c10600a347cbcde3d4).

Reply:
It seems that the error is because of the evaluate lib [this line](https://github.com/huggingface/evaluate/blob/81f841b3adebd2a78b3831388e74555acce61fa2/src/evaluate/evaluator/base.py#L20).